features/issues/components/issue-filters/issue-filters.tsx

```diff
@@ -0,0 +1,114 @@
+import { Button, Input, Option, Select } from "@features/ui";
+import styles from "./issue-filters.module.scss";
+import { useState } from "react";
+import { IssueLevel, IssueStatus } from "@api/issues.types";
+import { useRouter } from "next/router";
+
+export function IssueFilters() {
+  const [level, setLevel] = useState<Option | null>(null);
+  const [status, setStatus] = useState<Option | null>(null);
+  const [searchProject, setSearchProject] = useState<string>("");
```

Comment:
These states seem to cause a problem. When you refresh the page with filter params in the URL the values here aren't initialized. You can actually remove these states completely and only get the query params from the router. Then in the change handlers you directly update the query params without the need to store them in a state.